webapp/crashstats/supersearch/tests/test_views.py

```diff
@@ -105,209 +87,141 @@ def test_search_fields_metrics(self):
-                    "total": 4,
-                }
-                results["hits"] = self.only_certain_columns(
-                    results["hits"], params["_columns"]
-                )
-                return results
-
-            elif "SeaMonkey" in params.get("product", []):
-                results = {
-                    "hits": [
-                        {
-                            "signature": "nsASDOMWindowEnumerator::GetNext()",
-                            "date": "2017-01-31T23:12:57",
-                            "uuid": "663c9970-3f35-4b15-beaf-756d60220208",
-                            "product": "WaterWolf",
-                            "version": "1.0",
-                            "platform": "Linux",
-                            "build_id": 888981,
-                        },
-                        {
-                            "signature": "mySignatureIsCool",
-                            "date": "2017-01-31T23:12:57",
-                            "uuid": "1c6f67c8-a6ff-45ab-b5c5-4d2c20220208",
-                            "product": "WaterWolf",
-                            "version": "1.0",
-                            "platform": "Linux",
-                            "build_id": 888981,
-                        },
-                    ],
-                    "facets": {"build_id": [{"term": "888981", "count": 2}]},
-                    "total": 2,
-                }
-                results["hits"] = self.only_certain_columns(
-                    results["hits"], params["_columns"]
-                )
-                return results
-
-            elif "~nsASDOMWindowEnumerator" in params.get("signature", []):
-                results = {
-                    "hits": [
-                        {
-                            "signature": "nsASDOMWindowEnumerator::GetNext()",
-                            "date": "2017-01-31T23:12:57",
-                            "uuid": "ddb79c06-0d2f-4a7a-b2fc-d32a00220208",
-                            "product": "WaterWolf",
-                            "version": "1.0",
-                            "platform": "Linux",
-                            "build_id": 12345678,
-                        }
-                    ],
-                    "facets": {
-                        "signature": [
-                            {"term": "nsASDOMWindowEnumerator::GetNext()", "count": 1}
-                        ]
-                    },
-                    "total": 1,
-                }
-                results["hits"] = self.only_certain_columns(
-                    results["hits"], params["_columns"]
-                )
-                return results
-
-            else:
-                return {"hits": [], "facets": [], "total": 0}
-
-        SuperSearchUnredacted.implementation().get.side_effect = mocked_supersearch_get
+        def build_crash_data(**params):
+            crash_id = create_new_ooid()
+            data = {
+                "date_processed": utc_now(),
```

Comment:
Elasticsearch crash storage is date-sensitive. When redoing all these tests and taking the mocks out, I switched to using fresh crash ids. The `date_processed` is related to the crash id in that they're usually the same day, so I'm basing it on the same timestamp.